### ISSUE TITLE
So, ahem, how DO I add a guest to a team?

### ISSUE THREAD

**Issue by Rick-Kirkham** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#1687]
This article titled "Add a guest to a team" does not say **how to add a guest to a team**. :-(
Instead, it just gives high level background information about who can be added and who can do the adding.

So, could someone point me to instructions for adding a guest to a team? 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 93906e31-cd28-702d-bb7e-fcc2529df363
* Version Independent ID: afe4d6ed-cdcb-77e0-0951-f3c69e2463e1
* Content: [Add a guest to a team](https://docs.microsoft.com/en-us/microsoftteams/add-guests)
* Content Source: [Teams/add-guests.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/add-guests.md)
* Service: **msteams**
* GitHub Login: @somakbhattacharyya
* Microsoft Alias: **sbhatta**

**Comment by joshbarnhillF5** [comment]
https://www.youtube.com/watch?v=1daMBDyBLZc

**Comment by Rick-Kirkham** [comment]
That video is out-of-date. The UI has changed. As near as I can tell, there is no longer a Members tab. Even if the video worked, the information about how to add a guest should be in the article that is titled "Add a guest to a team".

**Comment by jmarks0981** [comment]
The video looks like the same UI I have, but when I type in guest names, it says it can't find that person.  There aren't clear directions.  Do I have to add them as an external contact in exchange first?  Do I add them somewhere else?  Is it a matter of waiting a few hours after allowing external and guest users (Yes, I know the difference) before I can see them?

Better documentation would be appreciated.

**Comment by scanum** [comment]
@officedocsbot assign @scanum

**Comment by sowjanya365** [comment]
Hi @Rick-Kirkham 

Thanks for reaching out! 

If you're a team owner, you can now invite guests to your team. Go to the team name, click **More options** > **click on the ellipsis button (...)** > **Add members**, and then enter the guest's email address.

**Notes:** 

Guests just need a valid email address.

If you receive a "We didn't find any matches" error while trying to add a guest, your organization doesn't allow guests. Contact your IT admin for more info.

For guests, you need to add the name other people will see. Click Edit guest information (pencil button) and then type their name. Be sure to take the time to add their name now—you’ll need the help of an IT admin to add it later.

Hope this clarifies your query. If not, would you be so kind to elaborate more on this.

Thanks.

**Comment by Rick-Kirkham** [comment]
@sowjanya365 
Thanks. But the crucial thing here is that you need to **add this information to the article**. Because currently you have an article titled "Add a guest to a team" that does _not_ say how to add a guest to a team!

**Comment by sowjanya365** [comment]
Hi @Rick-Kirkham 

Thanks for your valuable suggestions. We really appreciate them. 

The "Content Source" document is listed with what are all the ways to add guest to a team. And for the detailed explanation to add guests to a team you can refer to the link below.

https://support.office.com/en-gb/article/add-guests-to-a-team-fccb4fa6-f864-4508-bdde-256e7384a14f

Hope this helps. Kindly confirm.

Thank you.

**Comment by Rick-Kirkham** [comment]
@sowjanya365 
This is a not a good solution. You have an article on docs.microsoft.com titled "Add a guest to a team", which is actually about "_Ways to_ add a guest to a team", and which does _not_ contain practical steps for adding a guest to a team. Instead, you refer me to a _support_ article on support.office.com which does have those practical steps. And the first article doesn't even so much as link to the second one. What is the difference in purpose between these two pools of documentation?

**Comment by whitgroves** [comment]
I got the "we can't find any matches" error, but since I'm an admin I figured I could just go into Teams settings and allow guest access using this (https://docs.microsoft.com/en-us/microsoftteams/set-up-guests) guide. Well, I've enabled guest access in the organization but still get that error. Either 1 of 2 things are true: (a) it takes time for the updated setting to propagate, which should be in the documentation; or (b) there is another setting that needs to be changed, which should be in the documentation. Teams is supposed to make it easier to work with clients but at this point it seems to be wasting my org's time.
